pkgs/by-name/pa/papers/package.nix

```diff
@@ -46,19 +48,24 @@ stdenv.mkDerivation (finalAttrs: {
     owner = "GNOME/Incubator";
     repo = "papers";
     rev = finalAttrs.version;
-    hash = "sha256-T67d7xHK23CvmT8omEqNZrV5KloK4QXU973dtP9lTDE=";
+    hash = "sha256-kUPp79fv9jsFjViEWAAfE2oyEza++0wRhmWIB54ME7g=";
   };
 
-  cargoRoot = "shell-rs";
+  # FIXME: remove in next version
+  patches = [
+    (fetchpatch {
+      name = "fix-crash-when-drag-and-drop";
+      url = "https://gitlab.gnome.org/GNOME/Incubator/papers/-/commit/455ad2aebe5e5d5a57a2f4defc6af054927eac73.patch";
+      hash = "sha256-PeWlFhvM8UzUFRaK9k/9Txwgta/EiFnMRjHwld3O+cU=";
+    })
+  ];
 
-  cargoDeps = rustPlatform.importCargoLock {
-    lockFile = ./Cargo.lock;
+  cargoRoot = "shell-rs";
 
-    outputHashes = {
-      "cairo-rs-0.20.0" = "sha256-us8Q1cqHbs0wSfMcRgZx7wTqSagYkLv/aNI8Fle2nNk=";
-      "gdk4-0.9.0" = "sha256-a+fkiCilKbq7sBHZ9Uvq9a/qqbsVomxG6K07B5f4eYM=";
-      "libadwaita-0.7.0" = "sha256-gfkaj/BIqvWj1UNVAGNNXww4aoJPlqvBwIRGmDiv48E=";
-    };
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src pname version;
+    sourceRoot = "${finalAttrs.src.name}/${finalAttrs.cargoRoot}";
```

Comment:
https://github.com/NixOS/nixpkgs/pull/350541

Reply:
Please update

Reply:
thank you